core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairJobView.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecchronos.core.repair;
+
+import com.ericsson.bss.cassandra.ecchronos.core.repair.state.RepairStateSnapshot;
+import com.ericsson.bss.cassandra.ecchronos.core.utils.TableReference;
+
+/**
+ * Read only view of a scheduled repair job.
+ */
+public interface RepairJobView
+{
+    /**
+     * Return the table this job is scheduled for.
```

Comment:
I think it is better to use the @return tag, than writing "Return" in the description... For "simple" methods like this maybe no special "description" is needed. 
The @return tag is a mandatory tag for all non-void methods (https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return)

So what I'm trying to say is just change "Return" to "@return" :-)

(same for the other methods)

Reply:
Changing to "@return", I checked a bit on this and in the google coding guidelines this seemed to be preferred. But following oracle might be better.